### DISCUSSION TITLE
Some levelset I've made some time ago and never released

### DISCUSSION THREAD

**Discussion by DonAlgo** [Neverball/neverball#308] [Show and tell]
[Download the levelset](https://drive.google.com/file/d/1jCbcPYOUUMXdcpsOnahChttRseyVp7XY/view?usp=sharing)
![image](https://user-images.githubusercontent.com/82674433/197864222-9c7eee7e-b5d7-4359-bddc-79f0d156bee4.png)
![image](https://user-images.githubusercontent.com/82674433/197864244-2383cf24-cc93-4444-95d8-9287964077ab.png)
![image](https://user-images.githubusercontent.com/82674433/197864263-837774b3-4d05-4dd9-9faf-a05572be1a94.png)

I made **all the levels** exept from sb level 1 and level 2.

I had no idea that **switchball** was a game until I finished the first levelset
so some stuff might be wrong.

and the **template-switchbal**l folder is some stuff that came with the original levelset

hope you enjoy

**Comment by ersohnstyne** [comment]
This template is out of date, please consider to redownload all templates:
https://github.com/ersohnstyne/neverball-switchball-bundle-pack

**Reply by DonAlgo** [reply]
ok, thanks

**Reply by ersohnstyne** [reply]
PLUS:
- Placing the ball and goal outside the gyrocopter is **not allowed** on this level!
- Only one ball and goal can be used in campaign.
- The gyrocopter **MUST** attach at the front for ball **AND** attach at back for goal. Both attachment directions for gyrocopters are **not allowed**.
- Installing moving object requires **complex mechanics**, that you've imported from templates.
